src/main/kotlin/com/atlassian/performance/tools/report/api/judge/RelativeNonparametricStabilityJudge.kt

```diff
@@ -61,17 +61,17 @@ class RelativeNonparametricStabilityJudge(
                 action = action
             )
         val test = ShiftedDistributionRegressionTest(baseline, experiment, mwAlpha = 0.0, ksAlpha = significance)
-        return if (!test.equalDistributionsAfterShift) {
-            val message = "[$label] distribution shapes are different at $significance significance level"
+        return if (test.equalDistributionsAfterShift) {
             ActionReport(
-                report = FailedActionJunitReport(reportName, message),
-                action = action,
-                nonExceptionalFailure = true
+                report = SuccessfulJUnitReport(testName = reportName),
+                action = action
             )
         } else {
+            val message = "[$label] distribution shapes are different at $significance significance level"
```

Comment:
I don't understand this enough yet to propose a better message.
What does it mean for the log reader?

Reply:
I also don't understand yet. We cannot explain it to the user until we read up on the [KS](https://en.wikipedia.org/wiki/Kolmogorov%E2%80%93Smirnov_test) and [MWU](https://en.wikipedia.org/wiki/Mann%E2%80%93Whitney_U_test).

Reply:
According to wiki, the KS statistic is this black arrow:
![image](https://user-images.githubusercontent.com/1194704/195851875-7e360e78-db75-4cfa-aa23-e17a2e3c89db.png)
So the biggest vertical jump across the distros. We usually have nice smooth distros, which shoot up vertically at P100 (the max), so probably that's where it's detecting "the regression". It's too sensitive.

Reply:
Isn't that an assertion we want to remove?
https://ecosystem.atlassian.net/browse/JPERF-845